tests/test_daemon.cpp

```diff
@@ -109,6 +109,11 @@ struct Daemon : public mpt::DaemonTestFixture
             return mock_utils.Utils::filesystem_bytes_available(data_directory);
         });
 
+        ON_CALL(mock_utils, make_dir(_, _, _))
+            .WillByDefault([this](const QDir& a_dir, const QString& name, QFileDevice::Permissions permissions) {
+                return mock_utils.Utils::make_dir(a_dir, name, permissions);
+            });
```

Comment:
This is a big hammer, it affects all daemon tests. I understand you're just recovering what happened before, but now that we have the mock we might as well use it and prevent inadvertent filesystem changes affecting other tests. 

I had a look at the two tests that failed without this. Here's an alternative that fixes them individually: https://paste.ubuntu.com/p/XczdJ7MX9V/

Reply:
I see your point, but I'd rather just move the bypass to the specific test then rewrite a simplified version of the `make_dir()` function in a lambda.

Reply:
Sure, makes sense.